test/WebJobs.Script.Tests/Security/SecretManagerTests.cs

```diff
@@ -54,6 +55,88 @@ public SecretManagerTests(ITestOutputHelper outputHelper)
             _startupContextProvider = new StartupContextProvider(_testEnvironment, loggerFactory.CreateLogger<StartupContextProvider>());
         }
 
+        [Fact]
+        public void SecretManager_GenerateIdentifiableSecret_HonorsSeed()
```

Comment:
>SecretManager_GenerateIdentifiableSecret_HonorsSeed [](http://example.com/codeflow?start=20&length=51)

I've been conservative about updating tests until we could have a preliminary conversation on the key grouping (i.e., the # of seeds used to bucket allocations). Once that's decided, several of these tests could be updated to exercise the key generation logic and to verify that populated values have the expected signature/checksum.

Reply:
Agree that it would be good to update a few of the existing tests that add/update keys by adding a verification step at the end ensuring the right seed was used, the secrets are identifiable, etc. I.e. something that would catch a mistake if someone used the wrong seed value.

Reply:
Yes, makes sense. I will first merge this branch from main and then improve some tests. Are we all agreed that we're keeping the 3-seed breakdown, then? Master, system, and function?

Reply:
Yes, I'm fine with that. If @fabiocav / @brettsam don't have any opinions on this, I say go with what you have.

Reply:
ACK on this and your other notes, which I'm cleaning acting on today. Thanks for the advice.

Reply:
OK, so I have acted on your note, to update some tests to provide the validation you describe. I also made an effort to provide helpers that route people to the correct seeds.